Makefile

```diff
@@ -2,17 +2,18 @@
 .DEFAULT_GOAL := docker
 
 init:
-	clear
-	cd website && npm install
-	cd api && npm install
-	cd website/server && npm install
+	@cd website && npm install
+	@cd api && npm install
+	@cd website/server && npm install
+	@cd website && npm run build
+
+dev-up:
+	@docker compose -f docker-compose.dev.yml up -dV
 
-npm:
-	clear
-	cd api && npm run server
+dev-down:
```

Comment:
Are we replacing manually running `docker compose down` with `dev down`? And likewise for `dev-up`?

Reply:
Do we need to update the README for instructions on starting the app?

Reply:
Yep we're replacing those, and definitely, but I need to refactor the entire readme, so I'm going to do that in GV-62